src/atoms/Card/Card.stories.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+export default (asStory) => {
+  asStory('ATOMS/Card', module, (story, { Card }) => {
+    story
+      .add('default', () => (
```

Comment:
We already have the component something like that - `Paper`. It is various components or we can merge their? Or we should delete one of it?

Reply:
in my submission Paper has not Header or Footer

Reply:
Then we can remove header, section and footer from Paper and use Paper for Card?